azurerm_logic_app/outputs.tf

```diff
@@ -0,0 +1,31 @@
+output "id" {
+  value = azurerm_logic_app_workflow.logic_app.id
+}
+
+output "name" {
+  value = var.name
+}
+
+output "resource_name" {
```

Comment:
Actually there is no need to expose the resource_name .....

Reply:
ok, removed